recipes/openms/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# useless default include directory that is silently added by the compiler packages "to help"...
+# it is not even added with -isystem https://github.com/AnacondaRecipes/aggregate/blob/master/clang/activate-clang%2B%2B.sh#L87
+USELESS="-I${PREFIX}/include"
+export CXXFLAGS=${CXXFLAGS//${USELESS}/}
+
+# Not sure if those are needed 
+export LIBRARY_PATH=${PREFIX}/lib
+export LD_LIBRARY_PATH=${PREFIX}/lib
+#export DYLD_LIBRARY_PATH=${PREFIX}/lib
+
+mkdir build
+cd build
+
+if [[ $(uname -s) == Darwin ]]; then
+  RPATH='@loader_path/../lib'
+else
+  ORIGIN='$ORIGIN'
+  export ORIGIN
+  RPATH='$${ORIGIN}/../lib'
+fi
+
+# not sure if needed. CMake should take care of that.
+LDFLAGS='-Wl,-rpath,${RPATH}'
```

Comment:
Correct, let's let, as a first pass, CMake do its job. I'm hoping it will resolve things.

Conda and CMake should be able to work together to avoid this kind of manual intervention

Reply:
Ok, will try without it.